api/src/test/java/org/openmrs/module/fhir2/api/translators/impl/ObservationTranslatorImplTest.java

```diff
@@ -199,21 +198,21 @@ public void toFhirResource_shouldConvertObsValueToValue() {
 	}
 	
 	@Test
-	public void toFhirResource_shouldConvertObsGroupToComponent() {
+	public void toFhirResource_shouldConvertObsGroupToHasMemberReferences() {
 		Obs observation = new Obs();
 		Obs childObs = new Obs();
 		childObs.setUuid(OBS_UUID);
 		observation.addGroupMember(childObs);
 		
-		Observation.ObservationComponentComponent component = new Observation.ObservationComponentComponent();
-		component.setId(OBS_UUID);
-		when(observationComponentTranslator.toFhirResource(childObs)).thenReturn(component);
+		Reference reference = new Reference();
+		reference.setId(OBS_UUID);
```

Comment:
This should look something like this:

```java
reference.setType("Observation").setReference("Observation/" + OBS_UUID);
```

I.e., the reference id is not the `Obs` uuid (since the reference is a different entity for the observation itself).

Reply:
Thanks for the review! I'll make those changes.